docs/rules/computed-property-return.md

```diff
@@ -0,0 +1,44 @@
+# Enforces that a return statement is present in computed property (computed-property-return)
```

Comment:
I'd call this rule `return-in-computed-property` to make it more descriptive. WDYT?